frontend/src/metabase/query_builder/components/SidebarContent/SidebarContent.tsx

```diff
@@ -27,6 +30,31 @@ type Props = {
   "data-testid"?: string;
 };
 
+const SidebarContentMain = ({ children, ...props }: BoxProps) => {
+  return (
+    <Box
+      className={`${SidebarContentS.SidebarContentMain} sidebar-content-main`}
```

Comment:
```suggestion
      className={cx(SidebarContentS.SidebarContentMain, "sidebar-content-main")}
```

Reply:
string interpolation is always faster than calling a function, but latter more readable 👍